### ISSUE TITLE
Crash on inspecting a node connected to the output of 'Path from mask'

### ISSUE THREAD

**Issue by Kyrela** [Ttanasart-pt/Pixel-Composer#64]
## Description

As title says.

https://github.com/Ttanasart-pt/Pixel-Composer/assets/44508673/fe60b73d-6d01-4a58-9c9d-6db549f2c02e

I've also been able to experience similar issues with other nodes that "Trim Path" (used in this example)

Error message:
```
==========  Crash log [PXC 1.17.1.2] [Windows] ==========

ERROR in
action number 1
of Draw Event
for object o_main:


index out of bounds request 39 maximum size is 39

========== Stack trace ==========

gml_Script_getPointDistance@anon@1168@Node_Path_From_Mask@node_path_from_mask (line 59)
gml_Script_getPointRatio@anon@2067@Node_Path_From_Mask@node_path_from_mask (line 69)
gml_Script_getPointRatio@anon@1990@Node_Path_Trim@node_path_trim (line 77)
gml_Script_drawOverlay@anon@553@Node_Path_Trim@node_path_trim (line 29)
gml_Script_drawNodeTools@Panel_Preview@panel_preview (line 1256)
gml_Script_drawContent@Panel_Preview@panel_preview (line 1742)
gml_Script_draw@anon@25943@PanelContent@panel_data (line 943)
gml_Script_drawPanel@Panel@panel_data (line 678)
gml_Script_draw@anon@10822@Panel@panel_data (line 411)
gml_Script_draw@anon@10822@Panel@panel_data (line 430)
gml_Script_draw@anon@10822@Panel@panel_data (line 430)
gml_Script_draw@anon@10822@Panel@panel_data (line 430)
gml_Script_draw@anon@10822@Panel@panel_data (line 430)
gml_Object_o_main_Draw_64 (line 43)


========= Crash log end =========
```

## To reproduce

- download [minimal_reproductible.zip](https://github.com/user-attachments/files/15794903/minimal_reproductible.zip), extract it and open `minimal_reproductible.pxc`
- Connect the node "Path from Mask" to the node "Trim Path" (using the input/output "Path")
- Click on the "Trim Path" node

## Expected

The inspection panels of the node opens.

## Result

The UI disappears and a crash warning is created each frame.

## Information

Pixel Composer 1.17.1.2
Steam AND Itch.io version (tested on both)
Windows 11 23H2
AMD Ryzen 7 7840HS w/ Radeon 780M Graphics
NVIDIA GeForce RTX 4070 Laptop

**Comment by Ttanasart-pt** [comment]
Thank you for reporting. This bug should be fixed in the 1.17.2 beta, I'd recommend testing it there.

**Comment by Kyrela** [comment]
Tested with 1.17.rc5 and 1.17.3, can confirm it works perfectly.
I should have read the 1.17.2 Beta devlog, didn't know it was already fixed, sorry for the inconvenience.
